openproblems/tools/normalize.py

```diff
@@ -52,3 +53,19 @@ def sqrt_cpm(adata: ad.AnnData) -> ad.AnnData:
     _cpm(adata)
     adata.X = scprep.transform.sqrt(adata.X)
     return adata
+
+
+@decorators.normalizer
+def preprocess_logCPM_1kHVG(adata: ad.AnnData) -> ad.AnnData:
```

Comment:
Can we refactor this to `log_cpm_hvg(adata: ad.AnnData, n_genes: int = 1000)` to a) be consistent with the naming style in the rest of the file and b) allow users to change n_genes?